addon/services/activity-tracking.ts

```diff
@@ -0,0 +1,72 @@
+import Service, { inject as service } from '@ember/service';
+import Configuration from '@upfluence/hyperevents/configuration';
+
+export type ActivityType = 'page_view' | 'button_click' | 'component_view';
+export type Activity = {
+  type: ActivityType;
+  action?: string;
+  path: string;
+  route: string;
+  engine?: string;
+  version?: string;
```

Comment:
do you have any idea could be populated? so far i dont see it being used

Reply:
We discussed this with Philippe and we had a hard time finding why the product team would find this information relevant. The route & path are already enough for a dev to see which engine is used.

Unless you have other usages of the activity tracking service ?
Let's discuss this vocally as well !